### ISSUE TITLE
[Quest] A Tangled Skein

### ISSUE THREAD

**Issue by LukasVolgger** [azerothcore/azerothcore-wotlk#3657]
##### CURRENT BEHAVIOUR:

During the quest ID 12555 you have to use the item "Tangled Skein Thrower" ID 38515 on a "Plague Sprayer" ID 28274 and it should count as 1 objective out of 5. **Actually it count's twice:** 

![image](https://user-images.githubusercontent.com/55922592/98447125-bbe6c580-2122-11eb-99ef-525b610beaff.png)

##### EXPECTED BLIZZLIKE BEHAVIOUR:

YouTube Video from 2008: https://www.youtube.com/watch?v=rRg5HmxHISc&ab_channel=wtbblue

- The objective should count 1 for each "Plague Sprayer"

##### STEPS TO REPRODUCE THE PROBLEM:

1. .go 4919.589844 -3295.689941 311.007996 571
2. .q rem 12555 & .q a 12555
3. use the "Tangled Skein Thrower" ID 38515 on a "Plague Sprayer" ID 28274
4. the quest objective count's twice instead of one

##### AC HASH/COMMIT:

AzerothCore rev. 9a9308afd16a+ 2020-11-04 16:30:40 +0000 (master branch) (Win64, Release)

##### OPERATING SYSTEM:

Windows 10 Pro x64

##### MODULES:

- Auctionhouse Bot
- Autobalance
- Solo LFG

##### OTHER CUSTOMIZATIONS:

None

**Comment by Hacaw** [comment]
I can confirm this issue.
+ This video has better quality: https://www.wowhead.com/quest=12555/a-tangled-skein#videos:id=76252
+ The tower should drop after killing it.

TESTING
---
Used the commands provided.

SPEC
---
COMMIT: 1b156ef7
OPERATING SYSTEM: Win10+Docker
MODULES / CUSTOMIZATION: None

**Comment by Helias** [comment]
Thanks for the test :+1:

**Comment by Hacaw** [comment]
Good job, @r0m1ntik, just double-checked this issue.
TL;DR: **It works**.

![image](https://user-images.githubusercontent.com/6963500/106497512-d1238400-64c6-11eb-84c4-f795239cc60f.png)


##### STEPS TO REPRODUCE THE PROBLEM:
Described by the author.

##### AC HASH/COMMIT: 
adcf3f7a

##### DATABASE VERSION: 
WORLD: 2021_02_01_00
AUTH: 2021_01_25_00
CHARACTERS: 2021_01_25_00

##### OPERATING SYSTEM:
Windows 10 - Docker for Desktop

##### EXTRA NOTES: 
Chromie-Client (Downloaded from 10-01-2021)
DBC & Maps - v9 - 1-12-2020

##### MODULES:
None

##### OTHER CUSTOMIZATIONS:
None
